buildpipeline/pipeline.json

```diff
@@ -298,11 +298,13 @@
           }
         },
         {
-          "Name": "DotNet-CoreFx-Trusted-Windows-NoTest",
+          "Name": "DotNet-CoreFx-Trusted-Windows",
           "Parameters": {
             "PB_Platform": "x64",
```

Comment:
I asked the same question of @AlexGhiondea , and he said we only wanted x64 at this time.